composer.json

```diff
@@ -12,7 +12,7 @@
     "require": {
         "php": "^7.2 || ^8.0",
         "symfony/console": "^4.4",
-        "sonata-project/doctrine-orm-admin-bundle": "^3.15",
+        "sonata-project/doctrine-orm-admin-bundle": "^3.15 || ^4",
```

Comment:
3 en 4 zijn door de signatuur van de diverse methodes in de admin absoluut niet compatible. Dit moet dus aangepast worden naar alleen `^4`